packages/canner/src/query/utils.js

```diff
@@ -269,10 +269,24 @@ function genDeclareArgs(args, variables) {
     })
     .map(key => {
       const argValue = args[key];
-      return `${key}: ${argValue}`
+      return `${key}: ${convertKeynameToCorrectCase(argValue)}`
```

Comment:
It seems we don't have the function `convertKeynamToCorrectCase` since we can just edit the existed function `typeKey`